### PR TITLE
Subctl benchmark - add verbose option

### PR DESCRIPTION
**Pull request by maayanf24**
Add a verbose flag to subctl benchmark cmds. default=false
Fixes issue #748

Signed-off-by: Maayan Friedman <maafried@redhat.com>

### CODE REVIEW

File: pkg/subctl/cmd/benchmark.go [hunk 1]
```diff
@@ -36,16 +37,19 @@ var (
...
+	addBenchmarkFlags(benchmarkThroughputCmd)
 
 	benchmarkCmd.AddCommand(benchmarkThroughputCmd)
 	benchmarkCmd.AddCommand(benchmarkLatencyCmd)
 	rootCmd.AddCommand(benchmarkCmd)
 }
 
+func addBenchmarkFlags(cmd *cobra.Command) {
+	cmd.PersistentFlags().BoolVar(&intraCluster, "intra-cluster", false, "Runs the test within a single cluster")
+	cmd.PersistentFlags().BoolVar(&verbose, "verbose", true, "Produce verbose logs during benchmark tests")
```
Comment (mangelajo) [inline pkg/subctl/cmd/benchmark.go:50]: We normally have verbose = false in other parts
```suggestion
	cmd.PersistentFlags().BoolVar(&verbose, "verbose", false, "Produce verbose logs during benchmark tests")
```

**Review by mangelajo — CHANGES_REQUESTED** [review]
To do the same as in verify I'd default verbose to false. (I tend to enable it always though..)

File: pkg/subctl/cmd/benchmark.go (hunk 1 above)
Comment (skitt) [reply pkg/subctl/cmd/benchmark.go:50]: Yes, see https://github.com/submariner-io/submariner-operator/issues/649 for general discussion.

**Review by skitt — APPROVED** [review]

File: pkg/subctl/benchmark/throughput.go [hunk 2]
```diff
@@ -9,7 +9,10 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func StartThroughputTests(intraCluster bool) {
+var benchmark_verbose bool
```
Comment (tpantelis) [inline pkg/subctl/benchmark/throughput.go:12]: I think we should only have  global var for this.  I suggest renaming this to `Verbose` and reference it in _pkg/subctl/cmd/benchmark.go_ and remove the `verbose` var there. Then no need to pass it around.

**Review by tpantelis — CHANGES_REQUESTED** [review]

**Review by tpantelis — APPROVED** [review]

**Review by mangelajo — APPROVED** [review]
Looks good, but now in conflict because of the go.mod files sorry

**Pull request merged**